jobs/ci-kubernetes-e2e-gke-cvm-1-5-cvm-1-7-upgrade-cluster.env

```diff
@@ -6,5 +6,10 @@ KUBE_GKE_IMAGE_TYPE=container_vm
 
 PROJECT=gke-up-c1-3-clat-up-clu
 
+localSkip1="Daemon\ set\ \[Serial\]\ should\ run\ and\ stop\ complex\ \daemon\ with\ node\ affinity"
+localSkip2="\[SchedulerPredicates\]"
+localSkip3="\[SchedulerPriorities\]"
+
+GINKGO_TEST_ARGS=--ginkgo.skip="${localSkip1}|${localSkip2}|${localSkip3}"
```

Comment:
I just skimmed through the dashboard to see which 1.5-x upgrade jobs the daemon set test is failing for.

1.7 jobs (dashboard tab names from `1.5-1.7-upgrade`):
(first is the one changed in this PR)
```
gke-cvm-1-5-cvm-1-7-upgrade-cluster
gke-cvm-1-5-gci-1-7-upgrade-cluster
gke-gci-1-5-cvm-1-7-upgrade-cluster
gke-gci-1-5-gci-1-7-upgrade-cluster
```

I guess the daemon set tests aren't running on the master for 1.5-1.7 upgrades (they are for 1.5-1.6 upgrades)? 

This also needs to be fixed for the 1.5-1.6 upgrade tests (probably not critical for 1.7 release, but should still be fixed). Looking at the `release-1.6-upgrade-skew` dashboard, these upgrade jobs have the failing daemon set test:
```
gce-1.5-1.6-upgrade-cluster
gce-1.5-1.6-upgrade-master
gke-container_vm-1.5-container_vm-1.6-upgrade-cluster
gke-container_vm-1.5-container_vm-1.6-upgrade-master
gke-container_vm-1.5-gci-1.6-upgrade-cluster
gke-container_vm-1.5-gci-1.6-upgrade-master
gke-gci-1.5-container_vm-1.6-upgrade-cluster
gke-gci-1.5-container_vm-1.6-upgrade-master
gke-gci-1.5-gci-1.6-upgrade-cluster
gke-gci-1.5-gci-1.6-upgrade-master
```